src/connection/ConnectionOptionsReader.ts

```diff
@@ -78,9 +78,16 @@ export class ConnectionOptionsReader {
      * todo: get in count NODE_ENV somehow
      */
     protected async load(): Promise<ConnectionOptions[]|undefined> {
+        let connectionOptions: ConnectionOptions|ConnectionOptions[]|undefined = undefined;
+
+        const fileFormats = ["env", "js", "ts", "json", "yml", "yaml", "xml"];
+
+        // Detect if baseFilePath contains file extension
+        const possibleExtension = this.baseFilePath.substr(this.baseFilePath.indexOf("."));
```

Comment:
its not reliable way of getting a file extension

Reply:
Good catch, thank you.  Would you consider `lastIndexOf` reliable?

Reply:
I was thinking you'll use some method that gets you file extension... Or yeah `lastIndexOf` at least.